### PR TITLE
cmd: refresh page

### PR DESCRIPTION
**Pull request by EmilySeville7cfg**
- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** latest

### CODE REVIEW

File: pages/windows/cmd.md [hunk 1]
```diff
@@ -7,17 +7,17 @@
 
 `cmd`
 
-- Execute a [c]ommand:
+- Execute specific [c]ommands:
 
-`cmd /c "{{command}}"`
+`cmd /c "{{echo cmd is executed}}"`
```
Comment (CleanMachine1) [inline pages/windows/cmd.md:12]:
```suggestion
`cmd /c "{{echo Hello World}}"`
```

File: pages/windows/cmd.md [hunk 2]
```diff
@@ -7,17 +7,17 @@
...
+- Execute a specific script:
 
-`cmd {{path/to/file.bat}}`
+`cmd {{path/to/script.bat}}`
 
-- Execute a command and then enter an interactive shell:
+- Execute specific commands and then enter an interactive shell:
 
-`cmd /k "{{command}}"`
+`cmd /k "{{echo cmd is executed}}"`
```
Comment (CleanMachine1) [inline pages/windows/cmd.md:20]:
```suggestion
`cmd /k "{{echo Hello world}}"`
```

File: pages/windows/cmd.md (hunk 1 above)
Comment (CleanMachine1) [reply pages/windows/cmd.md:12]: I don't use windows, does this need to be in quotation marks since it is more than one word?

Comment (EmilySeville7cfg) [reply pages/windows/cmd.md:12]: They are not required (tested in wine cmd).

File: pages/windows/cmd.md [hunk 3]
```diff
@@ -7,17 +7,17 @@
 
 `cmd`
 
-- Execute a [c]ommand:
+- Execute specific [c]ommands:
 
-`cmd /c "{{command}}"`
+`cmd /c {{echo cmd is executed}}`
```
Comment (CleanMachine1) [inline pages/windows/cmd.md:12]:
```suggestion
`cmd /c {{echo Hello World}}`
```

File: pages/windows/cmd.md [hunk 4]
```diff
@@ -7,17 +7,17 @@
...
+- Execute a specific script:
 
-`cmd {{path/to/file.bat}}`
+`cmd {{path/to/script.bat}}`
 
-- Execute a command and then enter an interactive shell:
+- Execute specific commands and then enter an interactive shell:
 
-`cmd /k "{{command}}"`
+`cmd /k {{echo cmd is executed}}`
```
Comment (CleanMachine1) [inline pages/windows/cmd.md:20]:
```suggestion
`cmd /k {{echo Hello World}}`
```

File: pages/windows/cmd.md (hunk 2 above)
Comment (EmilySeville7cfg) [reply pages/windows/cmd.md]: Fix was applied.

**Review by CleanMachine1 — APPROVED** [review]

**Review by Waples — APPROVED** [review]
lgtm.

**Pull request merged**